packages/gatsby-plugin-less/README.md

```diff
@@ -47,6 +47,8 @@ plugins: [
 ]
 ```
 
+*This particular change would restore Gatsby v1 CSS modules behavior.*
```

Comment:
Do you think this is important to note?

Reply:
(and it's becoming more and more unimportant as more people use Gatsby v2!; so I'm inclined to say just scrap this line!)

Reply:
Happy to scrap. If so, I should probably elaborate within get migration doc.

Reply:
Yup, I'd move this to migration guide